pkg/openstack/instances.go

```diff
@@ -613,17 +634,47 @@ func nodeAddresses(srv *servers.Server, interfaces []attachinterfaces.Interface,
 	}
 
 	// process the rest
-	type Address struct {
-		IPType string `mapstructure:"OS-EXT-IPS:type"`
-		Addr   string
-	}
-
 	var addresses map[string][]Address
 	err := mapstructure.Decode(srv.Addresses, &addresses)
 	if err != nil {
 		return nil, err
 	}
 
+	// Get the private networks that are not directly connected
+	if len(addresses) < len(allPrivates) {
+		extraPrivates := make(map[string][]Address)
+		// For each private network
+		for k, v := range allPrivates {
+			ok := false
```

Comment:
Suggest ok -> found, because that's what it means here.

Reply:
comment taken